chat/v1/stream.proto

```diff
@@ -13,6 +13,8 @@ import "profile/v1/stream.proto";
 option go_package = "github.com/harmony-development/legato/gen/chat/v1";
 
 // Request type for use in the `StreamEvents` endpoint.
+// By default, this endpoint will subscribe to all events.
```

Comment:
Also mention that future guilds joined by the user will get added automatically too.